01-main/packages/nomad

```diff
@@ -1,4 +1,5 @@
 DEFVER=1
+ARCHS_SUPPORTED="amd64 arm64"
```

Comment:
Hashicorp seem to cover all releases but also seem to support amd64 so added that in passing